db/migrate/20151125155213_create_middleware_servers.rb

```diff
@@ -1,14 +1,17 @@
 class CreateMiddlewareServers < ActiveRecord::Migration
```

Comment:
I am aware of the issue per se, but was of the (false) assumption that you had a separate db for this.
I can create a new migration